.github/workflows/build.yml

```diff
@@ -5,8 +5,23 @@ on:
   pull_request:
 jobs:
   validation:
-    name: "Gradle Wrapper Validation"
+    name: "Build"
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: gradle/wrapper-validation-action@v1
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Install Java 17
+        uses: actions/setup-java@v3.9.0
+        with:
+          java-version: 17 # Can be more specific if neccesary
+          distribution: adopt
+      - name: Build
+        run: ./gradlew build
+      - name: Upload jars
+        uses: actions/upload-artifact@v3
+        with:
+          name: jars
+          path: build/libs
+          if-no-files-found: error
```

Comment:
Should be "warn" instead of error because otherwise, it would always assume compilation failed.

Reply:
Fixed, but I used error so that if building jars didn't produce any jars, the build would tell you that something is wrong.